reportTemplatesSrc/components/OnTrackToGraduateTable.js

```diff
@@ -0,0 +1,118 @@
+const React = require('react');
+const { first, capitalize, chunk, truncate } = require('lodash');
```

Comment:
The common pattern used in a lot of the projects is importing individual lodash functions instead of functions from the full build. 

Reason: something, something, smaller package size.